### PR TITLE
chore: add pre-install dotnet yml content for azure Func

### PR DESCRIPTION
**Pull request by frankqianms**

### CODE REVIEW

**Review by blackchoey — DISMISSED** [review]
Please also add a test case to test plan for this

File: packages/fx-core/templates/core/v3Migration/js.ts.app.yml [hunk 1]
```diff
@@ -116,6 +116,9 @@ deploy:
   {{/if}}
 {{/if}}
 {{#if activePlugins.fx-resource-function}}
+  - uses: prerequisite/install # Install dependencies
+    with:      
+      dotnet: true
   - uses: cli/runNpmCommand
     with:
       workingDirectory: api
```
[the remark this answers is not in the document]
Comment (frankqianms) [reply packages/fx-core/templates/core/v3Migration/js.ts.app.yml:124]: fixed

**Review by Siglud — APPROVED** [review]

File: packages/fx-core/templates/core/v3Migration/js.ts.app.yml [hunk 2]
```diff
@@ -124,6 +127,7 @@ deploy:
...
       args: build extensions.csproj -o bin --ignore-failed-sources
+      execPath: $\{{ {{~dotnetPath~}} }}
```
Comment (blackchoey) [inline packages/fx-core/templates/core/v3Migration/js.ts.app.yml:130]: Let's add a comment like: "Use dotnet installed by prerequisite/install action" to help users better understand this parameter

Comment (frankqianms) [reply packages/fx-core/templates/core/v3Migration/js.ts.app.yml]: added

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**